contracts/PrizeLinkedAccountVault.sol

```diff
@@ -287,8 +300,9 @@ contract PrizeLinkedAccountVault is
                 result[t] = previousDrawParticipants[i];
                 t++;
```

Comment:
This could be changed for ++t; and wrap in unchecked

Reply:
it is an external view btw.

Reply:
but I will change, to ++t to make it consistent

Reply:
Oups 😅